README.md

```diff
@@ -15,7 +15,7 @@ examples on how to use them.
 ### Quality checks
 
 __Description__:    This workflow runs automated checks, that test for compliance with our [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
-__Workflow file__:  [.github/workflows/quality-checks.yaml](.github/workflows/quality-checks.yaml)
+__Workflow file__:  [.github/workflows/reusable-quality-checks.yaml](.github/workflows/reusable-quality-checks.yaml)
```

Comment:
I think this is already fixed on main and is now causing a conflict

Reply:
thanks for the hint, adapted current changes in main branch to my Pull Request